### PR TITLE
Implement Deep Averaging Networks (DAN)

### PR DESCRIPTION
**Pull request by eyal-str**
Implement Deep Averaging Networks (DAN) - http://cs.umd.edu/~miyyer/pubs/2015_acl_dan.pdf

### CODE REVIEW

**Comment by RamaneekGill** [comment]
Just curious, whats the difference between deep averaging networks and FastText with dropout training and (deep) affine layers at the end of the network?

**Comment by eyal-str** [comment]
Mainly because of the hidden layers. I tried with 5 hidden layers 1 epoch.
It took 9 seconds with week gpu to finish and I got 87% accurecy on the
test set.  Ive just implemented it... but I think it is really amazing that
machine can do this magic in 9 seconds and the code is very short and easy
to understand. In my example I used only 3 layers and 2 epochs so it took
18 seconds to achieve the same results.

בתאריך 27 בינו׳ 2017 05:18,‏ "Ramaneek Gill" <notifications@github.com> כתב:

**Comment by fchollet** [comment]
Please consider submitting to `contrib` instead: https://github.com/farizrahman4u/keras-contrib
